src/components/details/details.yaml

```diff
@@ -1,3 +1,37 @@
+params:
+- name: summaryText
+  type: string
+  required: true
+  description: If `summmaryHtml` is set, this is not required. Text to use within the summary element (the visible part of the details element). If `summaryHtml` is provided, the `summaryText` argument will be ignored.
+- name: summaryHtml
+  type: string
+  required: true
+  description: If `summmaryText` is set, this is not required. Text or HTML to use within the summary element (the visible part of the details element). If `summaryHtml` is provided, the `summaryText` argument will be ignored.
+- name: text
+  type: string
+  required: true
+  description: If `html` is set, this is not required. Text or HTML to use within the disclosed part of the details element. If `html` is provided, the `text` argument will be ignored.
+- name: html
+  type: string
+  required: true
+  description: If `text` is set, this is not required. Text or HTML to use within the disclosed part of the details element. If `html` is provided, the `text` argument will be ignored.
```

Comment:
Suggest we change 'Text or HTML' to just 'HTML' to reduce cases where people are using the HTML option unnecessarily?